protected/widgets/Section.php

```diff
@@ -95,6 +107,17 @@ private function initCss(): void
 
     public function run(): string
     {
+        if (isset($this->permission)) {
+            if (!isset($this->subject)) {
+                throw new InvalidConfigException('To use permission, a subject must be set.');
+            }
+
+            if (!\Yii::$app->user->can($this->permission, $this->subject)) {
```

Comment:
If we use this service / component in multiple places we should use a private getter. (For easier refactoring later)